docs/element-definitions.md

```diff
@@ -1648,9 +1654,52 @@ series of bytes.
 }
 ```
 
-This reads, "The location starts at the 5th byte of the source file. It
-includes the 12 subsequent bytes including the starting one. Then it
-continues at the 21st byte for another 12 bytes."
+This reads, "The location starts at the 5th byte (the 2nd byte of line 3) of the source file".
+
+#### Example
+
+```json
+{
+  "element": "sourceMap",
+  "content": [
+    {
+      "element": "array",
+      "content": [
+        {
+          "element": "number",
+          "attributes": {
+            "line": {
+              "element": "number",
+              "content": 3
+            },
+            "column": {
+              "element": "number",
+              "content": 2
+            }
+          },
+          "content": 4
+        },
+        {
+          "element": "number",
+          "attributes": {
+            "line": {
+              "element": "number",
+              "content": 3
+            },
+            "column": {
+              "element": "number",
+              "content": 10
+            }
+          },
+          "content": 12
+        }
+      ]
+    }
+  ]
+}
+```
+
+**NOTE** *`line` and `column` are optional and may not always be available*.
```

Comment:
I'd suggest we roll this out only on the source maps for annotations at first. We need to introduce gzip into our parsing service before adding this to every source map as this will increase resultant document producing a large number of 509s.